notebooks/lightning_segmentation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>IMO: we can keep a version of this notebook where we build our own UNet (for our tutorials) but I think it would be really nice to have an "official" example with Monai's segmentation pipeline. (this would include model, loss, metric)</p><p>I think it is a step in the right direction to use standardised stuff like in the natural image domain. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>